config/redirects.yml

```diff
@@ -123,7 +123,6 @@ plugins:
       developer/serving/services/http-option.md: serving/services/http-protocol.md
       developer/serving/services/ingress-class.md: serving/services/ingress-class.md
       developer/serving/services/private-services.md: serving/services/private-services.md
-      developer/serving/services/service-metrics.md: serving/services/service-metrics.md
```

Comment:
I think you want to add redirects here to cover both the retired document and the earlier URL.  I'm not sure there's any disadvantage to keeping the earlier URLs working.